src/Components/WelcomePage/WelcomePage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const WelcomePage = () => {
+  return (
+    <section>
+      <h1>Hewwo</h1>
```

Comment:
This is honeslty feels like a great addition for a more interactive user experience. 
I am really enjoying how we are building a more cohesive concept and user path.

Reply:
Agreed! I love this intro page idea. I think it will make for a great user experience. I'm really excited to see what we create with this!